src/sdk/eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -331,18 +349,22 @@ jobs:
             echo "##vso[task.setvariable variable=additionalBuildArgs]--with-sdk /vmr/.dotnet"
           fi
 
-          docker run --rm -v "$(sourcesPath):/vmr" -w /vmr ${{ parameters.container }} ./prep-source-build.sh $customPrepArgs
+          cd $(sourcesPath)
```

Comment:
For the prep and build steps, would it make more sense to set workingDirectory to sourcesPath?

Reply:
Yes, that's a good idea